tests/conftest.py

```diff
@@ -1,4 +1,4 @@
-"""Copyright (c) 2017, Anthony Scopatz All rights reserved."""
+"""Copyright (c) 2017-2025, Anthony Scopatz All rights reserved."""
```

Comment:
Here let's change the copyright to Simon Billinge from 2021-2025.  for 2017-2020 it can be Anthony